crates/subspace-farmer/src/archiving.rs

```diff
@@ -192,15 +192,15 @@ impl Archiving {
                     maybe_result = best_block_number_receiver.next() => {
                         match maybe_result {
                             Some(Ok(Ok(best_block_number))) => {
-                                debug!("Best block number: {:#?}", best_block_number);
+                                debug!(best_block_number);
```

Comment:
Why text removed here?

Reply:
It will print field name and its value, so basically it will appear almost the same in logs